account_banking_reconciliation/__manifest__.py

```diff
@@ -5,13 +5,16 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 {
     "name": "Bank Account Reconciliation",
+    "summary": "Check transactions that cleared the bank",
     "version": "11.0.1.0.0",
     "license": "AGPL-3",
     "category": "Accounting and Financial Management",
-    "author": "NovaPoint Group LLC, Open Source Integrators, "
+    "author": "NovaPoint Group LLC, "
+              "Open Source Integrators, "
               "Odoo Community Association (OCA)",
     "website": "https://github.com/OCA/account-reconcile",
     "depends": [
+        "account_invoicing",
```

Comment:
Just a comment: Not sure about usage here but we are not using the reference of account_invoicing 
(As said by @pedrobaeza https://github.com/OCA/sale-workflow/pull/606#issuecomment-364025563 )